.github/workflows/ci.yml

```diff
@@ -43,6 +43,35 @@ jobs:
       - name: Run tests
         run: sh ci/run.sh ${{ matrix.os }}
 
+  create_issue:
+    runs-on: ubuntu-latest
+    # the combination of these two lines seems to do the "if build_and_test failed, run this one"
+    needs: build_and_test
+    if: always() && github.event_name == 'schedule' && needs.build_and_test.result == 'failure'
```

Comment:
So, the current condition won't prevent duplicate issues, i.e. it will create a new issue per day until we fix the failure?

Reply:
added a check while updating the code to use `gh` cli instead.